client/packages/system/src/Name/DetailModal/DetailModal.tsx

```diff
@@ -42,7 +42,10 @@ export const DetailModal: FC<DetailModalProps> = ({ nameId }) => {
           <DetailSection title="">
             <DetailInputWithLabelRow
               label={t('label.address')}
-              inputProps={{ value: data?.address, disabled: isDisabled }}
+              inputProps={{
+                value: [data?.address1, data?.address2].join(', '),
```

Comment:
Seems like address can be empty strings:

![image](https://user-images.githubusercontent.com/88299195/170364467-08b636a4-c9de-4600-b30c-bc83a3039060.png)

Is this a bug on the remote server side? I thought we interpret empty strings as null

Reply:
my testing is a little slack 🙄  Thought that I could sneak in a one-liner to concat addresses safely.. will revisit that.